wallet/src/main/java/com/android/identity_credential/wallet/presentation/OpenID4VPPresentationActivity.kt

```diff
@@ -522,7 +522,9 @@ class OpenID4VPPresentationActivity : FragmentActivity() {
 
         val vpTokenByteArray = generateVpToken(consentFields, credentialToUse, trustPoint, authorizationRequest, sessionTranscript)
         Logger.i(TAG, "Setting vp_token: ${vpTokenByteArray.decodeToString()}")
-        val vpToken = Base64.UrlSafe.encode(vpTokenByteArray)
+        val vpToken = if (credentialToUse is MdocCredential) {
```

Comment:
Similar here: when(), and handle each case explicitly, in case other credential types need different encoding.

Reply:
Discussed offline. Defaulting to no encoding is fine.